components/widgets/menu/menu_items/menu_item.scss

```diff
@@ -20,8 +20,8 @@
         }
 
         span.icon {
-            width: 28px; // less 12px padding is 16px
-            height: 16px; // less 3px top padding is 16px;
+            width: 26px; // less 12px padding is 16px
+            height: 14px; // less 3px top padding is 16px;
```

Comment:
please dont change the base-component SCSS ... alter values in the implementing component